kvmagent/kvmagent/plugins/vm_plugin.py

```
@@ -3608,12 +3845,61 @@ def cbd_volume():
+        automatic_iothread_vq_mapping_plan = None
+        created_automatic_iothread_ids = []
+        created_scsi_controller_alias = None
+        if volume_needs_automatic_iothread_vq_mapping(volume):
+            used_iothread_ids = iothread_ids_from_iothread_info(VmPlugin.get_iothread_info(self.uuid))
+            automatic_iothread_vq_mapping_plan = make_iothread_vq_mapping_plan(volume, used_iothread_ids)
+
+        def cleanup_created_iothread_vq_mapping():
+            if created_scsi_controller_alias:
+                try:
+                    self.detach_controller_by_alias(self.uuid, created_scsi_controller_alias)
+                except Exception as exc:
```

Ruff (0.15.13)

[warning] Do not catch blind exception: `Exception`

(BLE001)